recipes/libsemigroups-python-bindings/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   # branch instead
   #git_url: {{ repo }}.git
   #git_tag: python-bindings
-  url: {{ repo }}/archive/python-bindings.tar.gz 
+  url: {{ repo }}/archive/conda-test.tar.gz 
```

Comment:
I know. This wasn't an answer to your question. Your question and my comment were seconds apart